src/openrct2-ui/UiContext.Linux.cpp

```diff
@@ -251,10 +253,15 @@ namespace OpenRCT2::Ui
             return result;
         }
 
-        bool HasFilePicker() const override
+        bool HasFilePicker() override
```

Comment:
Think you would normally mark the cache as mutable rather than change the const of the function

Reply:
Sounds better, how can I do that?

Reply:
https://en.cppreference.com/w/cpp/language/cv 
Just add mutable before the `std::optional<bool> _hasFilePicker = std::nullopt;`